packages/components/src/auto-complete/auto-complete.board.tsx

```diff
@@ -1,17 +1,13 @@
-import React from 'react';
 import { createBoard } from '@wixc3/react-board';
-import { ZeeRoot } from '../simulation-mixins/mixin-zee-root';
+import React from 'react';
+import type { ItemData } from '../board-assets';
+import { createItems, ItemRenderer } from '../board-assets';
+import { mixinProjectThemes } from '../board-mixins/mixin-project-themes';
+import { ZeeRoot } from '../board-mixins/mixin-zee-root';
 import { AutoComplete } from './auto-complete';
```

Comment:
we like having an import order of:
1. 3rd party deps
2. monorepo packages
3. relative
4. style
5. type

this is only a recommendation